shopsdk/install.py

```diff
@@ -0,0 +1,322 @@
+	  b2 = "printful_execute_all\n"
+	  b3 = "digital_use_private_key\n"
+	  b4 = "digital_execute_all\n"
+	  b5 = "nwcs_use_private_key\n"
+	  b6 = "nwcs_execute_all\n"
+	  b7 = "accf_use_private_key\n"
+	  b8 = "accf_execute_all\n"
+	  b9 = "pwinty_production\n"
+	  c0 = "pwinty_send_stats\n"
+	  c1 = "pwinty_send_logs\n"
+	  c2 = "pwinty_load_options_file\n"
+	  c3 = "pwinty_use_pickle\n"
+	  c4 = "photos_production\n"
+	  c5 = "photos_send_stats\n"
+	  c6 = "photos_send_logs\n"
+	  c7 = "photos_load_options_file\n"
+	  c8 = "photos_use_pickle\n"
+	  c9 = "digital_production\n"
+	  d0 = "digital_send_stats\n"
+	  d1 = "digital_send_logs\n"
+	  d2 = "digital_load_options_file\n"
+	  d3 = "digital_use_pickle\n"
+	  d4 = "printful_production\n"
+	  d5 = "printful_send_stats\n"
+	  d6 = "printful_send_logs\n"
+	  d7 = "printful_load_options_file\n"
+	  d8 = "printful_use_pickle\n"
+	  d9 = "nwcs_production\n"
+	  e0 = "nwcs_send_stats\n"
+	  e1 = "nwcs_send_logs\n"
+	  e2 = "nwcs_load_options_file\n"
+	  e3 = "nwcs_use_pickle\n"
+	  e4 = "accf_production\n"
+	  e5 = "accf_send_stats\n"
+	  e6 = "accf_send_logs\n"
+	  e7 = "accf_load_options_file\n"
+	  e8 = "accf_use_pickle\n"
+	  e9 = "pwinty_sentry_url\n"
+	  f0 = "photos_sentry_url\n"
+	  f1 = "printful_sentry_url\n"
+	  f2 = "digital_sentry_url\n"
+	  f3 = "nwcs_sentry_url\n"
+	  f4 = "accf_sentry_url\n"
+	  f5 = "printful_cdn_url\n"
+	  f6 = "pwinty_cdn_url\n"
+	  f7 = "accf_cdn_url\n"
+	  f8 = "oc_url_root\n"
+	  f9 = "datadog_api_key\n"
+	  g0 = "pwinty_app_key\n"
+	  g1 = "photos_app_key\n"
+	  g2 = "printful_app_key\n"
+	  g3 = "digital_app_key\n"
+	  g4 = "pwinty_api_key\n"
+	  g5 = "printful_api_key\n"
+	  g6 = "oc_url_hash\n"
+	  g7 = "main_oc_username_hash\n"
+	  g8 = "main_oc_encryption_hash\n"
+	  g9 = "main_oc_directory_hash\n"
+	  h0 = "main_oc_primary_file_hash\n"
+	  h1 = "main_oc_secondary_file_hash\n"
+	  h2 = "main_oc_password_hash\n"
+	  h3 = "main_oc_primary_local_hash\n"
+	  h4 = "main_oc_secondary_local_hash\n"
+	  h5 = "secn_oc_username_hash\n"
+	  h6 = "secn_oc_encryption_hash\n"
+	  h7 = "secn_oc_directory_hash\n"
+	  h8 = "secn_oc_primary_file_hash\n"
+	  h9 = "secn_oc_secondary_file_hash\n"
+	  i0 = "secn_oc_password_hash\n"
+	  i1 = "secn_oc_primary_local_hash\n"
+	  i2 = "secn_oc_secondary_local_hash\n"
+	  i3 = "options_oc_username_hash\n"
+	  i4 = "options_oc_encryption_hash\n"
+	  i5 = "options_oc_directory_hash\n"
+	  i6 = "options_oc_primary_file_hash\n"
+	  i7 = "options_oc_secondary_file_hash\n"
+	  i8 = "options_oc_password_hash\n"
+	  i9 = "options_oc_primary_local_hash\n"
+	  j0 = "options_oc_secondary_local_hash\n"
+	  j1 = "othr_oc_username_hash\n"
+	  j2 = "othr_oc_encryption_hash\n"
+	  j3 = "othr_oc_directory_hash\n"
+	  j4 = "othr_oc_primary_file_hash\n"
+	  j5 = "othr_oc_secondary_file_hash\n"
+	  j6 = "othr_oc_password_hash\n"
+	  j7 = "othr_oc_primary_local_hash\n"
+	  j8 = "othr_oc_secondary_local_hash\n"
+	  j9 = "accf_oc_username_hash\n"
```

Comment:
The amount of redundant variables is terrible.

Reply:
Yeah, I nuked a lot of the extra features.